specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/loadBalancer.json

```diff
@@ -1347,6 +1430,38 @@
       ],
       "description": "Frontend IP address of the load balancer."
     },
+    "LoadBalancerBackendAddressPropertiesFormat": {
+      "properties": {
+        "virtualNetwork": {
```

Comment:
Since I'm also working on Azure CLI, I have one more question. Based on your example, are `virtualNetwork` and `ipAddress` updatable?

Reply:
Can you check with @gmainar?